src/Orchard.Web/Modules/Orchard.Setup/Services/SetupService.cs

```diff
@@ -174,7 +177,7 @@ public class SetupService : Component, ISetupService {
                 }
             }
 
-            // Set shell state to "Initializing" so that subsequent HTTP requests are responded to with "Service Unavailable" while Orchard is setting up.
+            // Reset shell state to "Initializing" so that subsequent HTTP requests are responded to with "Service Unavailable" while Orchard is setting up.

```

Comment:
Yes exactly, I thought the exact same thing. It would break implementations if `IOrchardHost`, which most likely means we wouldn't be breaking anyone's code.